pyproject.toml

```diff
@@ -119,10 +119,7 @@ netaddr = "~0.8.0"
 # Library of network related Jinja Filters
```

Comment:
Please also evaluate whether it's possible and desirable to update to the latest versions of `GitPython` and `jsonschema`, as both are currently pinned to older versions due to the python 3.6 dependency.